### ISSUE TITLE
[connect modal] New version for MVP

### ISSUE THREAD

**Issue by jabby09** [KlimaDAO/bezos-frontend#196]
### Summary
The current version of the Connect Modal needs updating for Carbonmark and then there are other future enhancements based on both a new SDK and UX improvements to follow. This issue covers only what is required for Carbonmark MVP.

### Requirements

`** Must-have for launch **`

- Updated branding ([already in progress](https://www.figma.com/file/uNBByAKEShEf3ryOX3R3Q2/Login?node-id=1%3A1189&t=e333JANznvHlKrhj-0#369246654))
- Move social login to top of modal, followed by wallets, followed by new Institutional option with the [Circle logo](https://www.circle.com/en/)
- Note...clicking on the Institutional option will initially direct the user to an article page (link to be provided)

`** Stretch goal for launch (let's design for this so it is captured) **`

- Update the visual style of the social logins so that it is clear they are one button to trigger the torus modal
- On that button, or in the over line text there, can we indicate that the social logins are powered by torus, or show the torus logo, or something

** `Questions` **

1. Most of the changes above apply to the connect modal on KlimaDAO app, the one question would be Institutional

** `Future Considerations` **

Once MVP is complete, we will want to consider rebuilding the modal using the new web3auth SDK. Self host option, we provide the UI design, fully customizable (https://github.com/tkey/tkey/)
